main.py

```diff
@@ -219,21 +219,21 @@ def webhook():
             return jsonify({"error": "Failed to get GitHub token"}), 500
 
         if event == 'issues' and data['action'] == 'opened':
-            issue = data['issue']
-            repo = data['repository']
-            owner = repo['owner']['login']
-            repo_name = repo['name']
-
-            result, status_code = create_pull_request_for_issue(token, owner, repo_name, issue)
+            result, status_code = create_pull_request_for_issue(
```

Comment:
I like how we are using named parameters when we call the main functions, but I don't like the look of having 'kwargs' dicts everywhere. Can we use named parameters but still have access to the parameters as regular variables as we did before?

Reply:
This isn't really using named parameters:
```
 result, status_code = create_pull_request_for_issue(
                token,
                data['repository']['owner']['login'],
                data['repository']['name'],
                data['issue']
            )
```

I expect to be able to do something like this:
```
 result, status_code = create_pull_request_for_issue(
                token=token,
                owner=data['repository']['owner']['login'],
                repo_name=data['repository']['name'],
                issue=data['issue']
            )
```

But don't want to see 'kwargs' dicts everywhere. Can we do something similar to destructuring objects like you can do in JavaScript?

Reply:
This is on the right track, but don't use the dictionary unpacking thing, I think that just makes the code a bit harder to understand.